modules/atomic-widgets/module.php

```diff
@@ -51,6 +68,12 @@ public function __construct() {
 			add_filter( 'elementor/widgets/register', fn( Widgets_Manager $widgets_manager ) => $this->register_widgets( $widgets_manager ) );
 			add_filter( 'elementor/editor/localize_settings', fn( array $settings ) => $this->add_prop_types_config( $settings ) );
 			add_action( 'elementor/editor/after_enqueue_scripts', fn() => $this->enqueue_scripts() );
+
+			add_filter(
+				'elementor/atomic-widgets/styles/transformers',
+				fn ( array $transformers) => $this->register_style_transformers( $transformers )
+			);
+			add_action( 'elementor/element/parse_css', fn( Post $post, Element_Base $element ) => $this->parse_atomic_widget_css( $post, $element ), 10, 2 );
```

Comment:
Please move all the logic of the rendering to a different class, something like `Element_Parse_CSS`, the action could be here in the module but the logic can live outside

Reply:
Done